ccm_web/server/src/api/api.controller.ts

```diff
@@ -93,6 +113,17 @@ export class APIController {
 
   @UseInterceptors(InvalidTokenInterceptor)
   @ApiSecurity('CSRF-Token')
+  @Post('admin/createExternalUsers')
+  async createExternalUsers (
+    @Body() externalUsersData: ExternalUsersDto, @UserDec() user: User): Promise<ExternalUserData> {
+    const externalUsers: ExternalUserDto[] = externalUsersData.users
+
+    const result = await this.apiService.createExternalUsers(user, externalUsers)
+    if (!result.success) throw new HttpException(result.data, HttpStatus.BAD_GATEWAY)
+    return result.data
+  }
+
+  @UseInterceptors(InvalidTokenInterceptor)
```

Comment:
```suggestion
  @UseInterceptors(InvalidTokenInterceptor)
  @ApiSecurity('CSRF-Token')
```

Reply:
Are you **_certain_** about this change?  I vaguely remember a conversation with @ssciolla in which we determined it was not necessary.

Reply:
When I added that like, it started working, Since this is POST, CSFR token is needed. This call enrolling users to sections has nothing to do with Admin token. Should would.

Reply:
Yes, this is needed for mutating requests if you want it to work in the Swagger UI. The CSRF middleware isn't controlled by this, it's just part of getting Swagger to work for us.

I think this was accidentally removed at some point; it was there before. It looks weird in the diff because you added a new endpoint right above it.

Reply:
🏁